dwolla_v2.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 3.0"
   spec.add_development_dependency "webmock", "~> 1.22"
 
+  spec.add_dependency "public_suffix", "~> 1.4.6"
```

Comment:
Any chance you guys could use the recent version of public_suffix, anything >= 2.x? A number of popular gems (such as webmock) have a dependency tree that requires public_suffix >= 2.x and this makes using dwolla_v2 >= 1.2 not compatible.